src/reset_mol_ids.cpp

```diff
@@ -76,10 +77,14 @@ void ResetMolIDs::command(int narg, char **arg)
       offset = utils::tnumeric(FLERR,arg[iarg+1],1,lmp);
       if (offset < -1) error->all(FLERR,"Illegal reset_mol_ids command");
       iarg += 2;
+    } else if (strcmp(arg[iarg],"verbose") == 0) {
+      if (iarg+2 > narg) error->all(FLERR,"Illegal reset_mol_ids command");
+      verbose = utils::inumeric(FLERR,arg[iarg+1],1,lmp);
+      iarg += 2;
     } else error->all(FLERR,"Illegal reset_mol_ids command");
   }
 
-  if (comm->me == 0) utils::logmesg(lmp,"Resetting molecule IDs ...\n");
+  if (verbose == 1 && comm->me == 0) utils::logmesg(lmp,"Resetting molecule IDs ...\n");
```

Comment:
most consistent with the LAMMPS programming style would be to write this as: `if (verbose && (comm->me == 0)) ...`